src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,102 @@
+.icon {
+  display: block;
+  width: 100%;
+  height: 18px;
+
+  @include onTablet {
+    height: 24px;
+  }
+
+  &--phone {
+    @extend %bgImageNormalize;
+    background-image: url(../images/met_phone_icon.svg);
+
+    @include onDesktop {
+
+      &::before {
+        content: "The Met Fifth Avenue +1 212-535-7710";
+        position: relative;
+        top: 0;
+
+        display: block;
+        width: 210px;
+        padding: 0 20px 8px;
+
+        color: white;
+        text-align: right;
+        font-weight: 700;
+        font-size: 16px;
+        line-height: 27px;
+        text-transform: uppercase;
+        text-shadow: 18px 4px 26px rgba(0, 0, 0, 0.44);
+
+        transform: translate(-100%, -66.6%);
+        transition: opacity 0.5s;
+        opacity: 0;
+        pointer-events: none;
+      }
+
+      &:hover::before {
+        opacity: 1;
+        pointer-events: all;
+      }
+
+      &::after {
+        content: "The Met Cloisters +1 212-923-3700";
```

Comment:
Move hard-coded text 'The Met Cloisters +1 212-923-3700' into a separate variable or a data attribute to make it easier to manage and update.